src/pages/index.tsx

```diff
@@ -1,82 +1,102 @@
+          <div tw="py-12 pl-12 z-10">
+            <div tw="text-3xl font-bold text-white mb-4">Quickstart Guide</div>
+            <div tw="text-xl font-normal text-white mb-12 max-w-md">A short guide on making your first deploy on Railway in a couple of minutes.</div>
+            <ButtonLink
+              href="/getting-started"
+              tw="w-40 bg-pink-700 text-white text-center text-lg hover:bg-pink-500"
+            >
+              <div tw="flex flex-row justify-center items-center font-semibold">
+                <ArrowRight tw="mr-1" />
+                Get Started
+              </div>
 
-          <ButtonLink
-            href="https://railway.app"
-            css={[
-              tw`bg-pink-500 text-white border-2 border-pink-500`,
-              tw`hover:bg-pink-400`,
-              tw`focus:ring-2 focus:ring-pink-700`,
-            ]}
-          >
-            Visit Railway
-          </ButtonLink>
-        </div>
-
-        <div tw="prose mt-16">
-          <section>
-            <h3 tw="flex items-center space-x-4">
-              <RssIcon /> <span>Changelog</span>
-            </h3>
-            <p>
-              Weekly update with features and improvements made to Railway.{" "}
-              <br />
-              <Link href="https://railway.app/changelog">View Changelog</Link>
-            </p>
+            </ButtonLink>
+          </div>
+          <div tw="absolute -bottom-8 right-0 opacity-50 z-0 lg:opacity-100">
+            <NextImage
+              src="/images/code-terminal.svg"
+              alt="Graphic of Terminal"
+              tw=""
+              layout="intrinsic"
+              height={350}
+              width={310}
+              quality={100} />
+          </div>
 
-            <h3 tw="flex items-center space-x-4">
-              <TwitterIcon /> <span>Twitter</span>
-            </h3>
-            <p>
-              Keep up with the latest news and updates. <br />
-              <Link href="https://twitter.com/Railway">
-                Follow @Railway
-              </Link>
-            </p>
+        </div>
 
-            <h3 tw="flex items-center space-x-4">
-              <DiscordIcon /> <span>Discord</span>
-            </h3>
-            <p>
-              Chat with our Railway members, ask questions, hang out.
-              <br />
-              <Link href="https://discord.gg/xAm2w6g">Join Discord</Link>
-            </p>
-            <h3 tw="flex items-center space-x-4">
-              <GitHub /> <span>Contributing</span>
-            </h3>
-            <p>
-              Each page footer contains an "Edit on GitHub Link". Change the
-              markdown, make a pull request, and we'll merge it! Deploys will
-              happen automagically cause the docs are hosted on Railway.
-              <br />
-              <Link href="https://github.com/railwayapp/docs">Doc Source</Link>
-            </p>
-          </section>
+        <div tw="mt-8 grid grid-cols-1 gap-8 md:grid-cols-2">
```

Comment:
nit/suggestion: I think the gap should be a bit smaller on mobile.

![image](https://user-images.githubusercontent.com/3044853/133295864-60330698-ebdf-4dd1-94ab-5526924a88ac.png)
```suggestion
        <div tw="mt-8 grid grid-cols-1 gap-4 md:gap-8 md:grid-cols-2">
```